src/main/java/com/kamycz3q/coursesystemspringboot/personalData/personalDataObject/PersonalData.java

```diff
@@ -0,0 +1,25 @@
+package com.kamycz3q.coursesystemspringboot.personalData.personalDataObject;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.sql.Timestamp;
+
+@Getter
+@Setter
+@Entity
+public class PersonalData {
+    @Id
+    @GeneratedValue(
+            strategy = GenerationType.AUTO
+    )
+    private Long id;
+    private String name, surname, email;
+    private Timestamp birthDate;
+    private String city, address, postCode;
```

Comment:
Po pierwsze warto te pola wydzielić do dedykowanej klasy "Address". Następnie String address generalnie należałoby rozbić na to z czego w rzeczywistości się to pole składa, czyli pewnie ulica, nr budynku, nr lokalu. W ten sposób można przeprowadzić walidacje na niepustość ulicy i nru budynku z osobna, żeby mieć większą pewność że adres jest poprawny. Gdybyśmy musieli walidować te części składowe mając tylko jednego stringa, to trzeba by zastosować jakiegoś regexa, a szkoda procesora.